Engine/Module/ModuleScriptManager.cpp

```diff
@@ -75,6 +75,7 @@ long ModuleScriptManager::TimeStamp(const char* path)
 
 void ModuleScriptManager::GetCurrentPath()
 {
+	//TODO:Move this
```

Comment:
This function is to get the current path from the running machine, necessary to patch the dll and pdbs